source/includes/usage-examples/code-snippets/watch.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func main() {
+	var uri string
+	if uri = os.Getenv("MONGODB_URI"); uri == "" {
+		log.Fatal("You must set your 'MONGODB_URI' environmental variable. See\n\t https://docs.mongodb.com/drivers/go/current/usage-examples/")
+	}
+
+	client, err := mongo.Connect(context.TODO(), options.Client().ApplyURI(uri))
+	if err != nil {
+		panic(err)
+	}
+	defer func() {
+		if err = client.Disconnect(context.TODO()); err != nil {
+			panic(err)
+		}
+	}()
+
+	// begin watch
+    coll := client.Database("insertDB").Collection("haikus")
+    pipeline := mongo.Pipeline{bson.D{{"$match", bson.D{{"operationType", "insert"}}}}}
+    cs, err := coll.Watch(context.TODO(), pipeline)
+    defer cs.Close(context.TODO())
+
+    if err := cs.Err(); err != nil {
+        panic(err)
+    }
+
+    fmt.Println("Waiting For Change Events. Insert something in MongoDB!")
+	
+    for cs.Next(context.TODO()) {
+        var event bson.D
+        if err := cs.Decode(&event); err != nil {
+            panic(err)
+        }
+        fmt.Println(event[3])
```

Comment:
```suggestion
        fmt.Println(event[3].Value)
```

Suggestion: I think this can make the Output cleaner (removes outer braces and "fullDocument" string)